sdks/java/core/src/main/java/org/apache/beam/sdk/util/IOChannelUtils.java

```diff
@@ -202,4 +202,25 @@ public static IOChannelFactory getFactory(String spec) throws IOException {
   public static String resolve(String path, String other) throws IOException {
     return getFactory(path).resolve(path, other);
   }
+
+  /**
+   * Resolve given multiple {@code others} against the {@code path}.
```

Comment:
Resolve given multiple {@code others} against the {@code path}. -> Resolve multiple {@code others} against the {@code path} sequentially.